packages/calcite-components/src/components/link/link.scss

```diff
@@ -1,3 +1,13 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-link-text-color: Specifies the component's text color.
+ * @prop --calcite-link-border-color: Specifies the component's border color.
+ * @prop --calcite-link-icon-color: Specifies the icon color.
```

Comment:
link icon color should inherit from the link text color. If a user wants to change the design pattern they can apply a `--calcite-icon-color` to the link